src/fklearn/training/classification.py

```diff
@@ -6,6 +6,7 @@
 from sklearn.feature_extraction.text import TfidfVectorizer
 from sklearn.linear_model import LogisticRegression
 from sklearn import __version__ as sk_version
+from catboost import CatBoostClassifier
```

Comment:
You don't need this here, do you? You are importing inside each function. It may break the independence from catboost installation when doing such as fklearn[xgboost]